open_humans/templates/pages/terms.html

```diff
@@ -152,9 +152,9 @@ <h3>Copyright</h3>
     </p>
     <h3>Kids</h3>
     <p>
-      Sorry, but Open Humans is not directed to people 12 and younger. If we
+      Sorry, but Open Humans is not directed to people 15 and younger. If we
       become aware that we have directly collected personal information from a
-      child under 13, we will delete that information.
+      child under 16, we will delete that information.
```

Comment:
maybe "... child under 16 without a parent or guardian's authorization"?

Reply:
Authorization within US COPPA (which this partly covers) is a big headache, if I recall correctly. Which is probably why most websites just say "nope". Child data _can_ exist in Open Humans (and does, I expect), but the account has to be managed by a parent/guardian.

Reply:
Makes sense.